packages/frontend/src/components/wallet-migration/modals/CleanKeysModal/CleanKeysModal.jsx

```diff
@@ -33,11 +33,11 @@ async function getAccountDetails({ accountId, publicKeyBlacklist, wallet }) {
         }, {});
 
     const allAccessKeys = await wallet.getAccessKeys(accountId);
-    const publicKey = await wallet.getPublicKey(accountId);
+    const currentAccount = await wallet.getLocalAccessKey(accountId, allAccessKeys);
```

Comment:
Why change this again? `wallet.getPublicKey` should be returning the exact same thing, it would be best to use the same function throughout this workflow.

Reply:
must be mistake from rebasing, will replace back to getPublicKey